### PR TITLE
Changes to section summaries and return to hub link

### PR DESCRIPTION
**Pull request by berroar**
### What is the context of this PR?
Updates the section summary page so that the submit button shows `Continue` rather than `Save and Continue`

Removes the guidance: `Check your answers before continuing to the next section` from section summary pages.

Removes `Can't complete this question?` from the return to hub link text.

### How to review 
Check that the changes listed above have been implemented.

### Checklist

* [ ] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

**Comment by berroar** [comment]
[[A] Changes to Runner - Section summaries and return to hub link (S)](https://trello.com/c/bGAsIEQl/4001-a-changes-to-runner-section-summaries-and-return-to-hub-link-s)

File: templates/sectionsummary.html [hunk 1]
```diff
@@ -51,3 +46,15 @@ <h2>{{summary.title}}</h2>
       {% endfor %}
   {% endif %}
 {% endblock %}
+
```
Comment (petechd) [inline templates/sectionsummary.html:49]: Do we test this new button text on section summary anywhere?

Comment (berroar) [reply templates/sectionsummary.html:49]: We do now

File: tests/functional/spec/features/section_summary/section_summary.spec.js [hunk 2]
```diff
@@ -18,6 +18,10 @@ describe("Section Summary", () => {
       expect($(PropertyDetailsSummaryPage.insuranceTypeAnswer()).getText()).to.contain("Both");
     });
 
+    it("When I get to the section summary page, Then the submit button should read continue", () => {
```
Comment (JamesGardiner) [inline tests/functional/spec/features/section_summary/section_summary.spec.js:21]:
```suggestion
    it("When I get to the section summary page, Then the submit button should read 'Continue'", () => {
```

**Pull request merged**